demo/flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -339,7 +342,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
I believe our ios binding is iOS 13.0 minimum - we should change the deployment targets to 13.0 where applicable